lang-scala/src/main/scala/mutability/ImmutabilityCar.scala

```diff
@@ -0,0 +1,7 @@
+package mutability
```

Comment:
should we use a package structure with `baeldung` in it? - as per https://github.com/Baeldung/scala-tutorials/blob/master/scala-lang/src/main/scala/defvarval/Values.scala ?